tools/telemetry-generator/package.json

```diff
@@ -35,21 +35,22 @@
 		"applicationinsights": "^2.4.1"
 	},
 	"devDependencies": {
-		"@fluid-internal/mocha-test-setup": "^2.3.1",
-		"@fluidframework/build-common": "^2.0.1",
-		"@fluidframework/eslint-config-fluid": "^5.2.0",
+		"@fluid-internal/mocha-test-setup": "^2.11.0",
```

Comment:
Nit: would it be worth updating these deps in a separate PR?

Reply:
Since all the checks passed, I think there's little value in splitting it at this point, I'll keep them here.

Reply:
(It is fairly hard to migrate package managers and not get dep updates. So, I don't worry too much about getting more updates in one shot.)